lmdeploy/serve/async_engine.py

```diff
@@ -55,12 +54,10 @@ def _config_model_name(config):
         model_name = get_model_name_from_workspace_model(model_path)
     # may get a model name from model_path
     if model_name is None:
-        model_name = best_match_model(model_path)
-        if model_name is None:
-            raise ArgumentError(None,
-                                f'Please set model_name for {model_path}')
+        if os.path.exists(model_path):
```

Comment:
https://github.com/vllm-project/vllm/blob/e07aff9e52342dc82b73c803ba69601242801bc4/vllm/entrypoints/openai/api_server.py#L180-L181

Reply:
The condition line is above.

Reply:
I mean in vLLM the served_model_name is the model_path if it is not specified